tamars_cookies.md

```diff
@@ -0,0 +1,42 @@
+# Ingredients
+
+
+
+```
+45 grams all-purpose flour
+30 grams Dutch-processed cocoa powder
+¾ teaspoon baking powder
+½ teaspoon kosher salt
+225 grams bittersweet chocolate (70 percent), broken or chopped into pieces
+55 grams unsalted butter (1/2 stick)
+2 large eggs, at room temperature
+100 grams granulated sugar
+55 grams light brown sugar
+1 teaspoon vanilla extract
+115 grams bittersweet chocolate chips or chopped chocolate (about 2/3 cup)
```

Comment:
Consider adding bakers percentages following the format that we have in the challah recipe.

Reply:
Nahhhh